### PR TITLE
fixed url and add telemetry

### PR DESCRIPTION
**Pull request by albaloo**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/find/azext_find/custom.py [hunk 1]
```diff
@@ -95,6 +95,12 @@ def set_custom_properties(prop, name, value):
     if name and value is not None:
         # 10 characters limit for strings
         prop['{}{}'.format(FIND_EXTENSION_PREFIX, name)] = value[:10] if isinstance(value, str) else value
+        prop['{}{}'.format(FIND_EXTENSION_PREFIX, 'session_id')] = telemetry_core._session._get_base_properties()[  # pylint: disable=protected-access
+            'Reserved.SessionId']  # pylint: disable=protected-access
+        prop['{}{}'.format(FIND_EXTENSION_PREFIX, 'subscription_id')] = telemetry_core._get_azure_subscription_id()  # pylint: disable=protected-access
+        prop['{}{}'.format(FIND_EXTENSION_PREFIX, 'client_request_id')] = telemetry_core._session.application.data[   # pylint: disable=protected-access
+            'headers']['x-ms-client-request-id']  # pylint: disable=protected-access
+        prop['{}{}'.format(FIND_EXTENSION_PREFIX, 'installation_id')] = telemetry_core._get_installation_id()  # pylint: disable=protected-access
```
Comment (williexu) [inline src/find/azext_find/custom.py]: I don't believe you need to send these properties, as they are already sent with telemetry.

Comment (albaloo) [reply src/find/azext_find/custom.py]: Jason couldn't find them.

Comment (williexu) [reply src/find/azext_find/custom.py]: @JasonRShaver do we still need to send these properties? We talked briefly about using the correlation ids of the events to link them together. Were you able to look into it?

**Comment by williexu** [comment]
Closing this due to lack of activity, please create a new PR if needed

**Pull request closed without merging**